packages/web-pkg/src/components/ItemFilter.vue

```diff
@@ -72,6 +73,16 @@ export default defineComponent({
       type: String,
       required: true
     },
+    placeholder: {
+      type: String,
+      required: false,
+      default: ''
+    },
+    label: {
+      type: String,
+      required: false,
+      default: ''
+    },
```

Comment:
I see your point and I think it's valid from a "inside the component"-perspective. On the other hand, when using the component ("outside the component perspective"), there is only one label and one placeholder. My thinking was, why make it unnecessary complex then?

Reply:
> On the other hand, when using the component ("outside the component perspective"), there is only one label and one placeholder.

Well, not necessarily, because that's strictly bound to the `showFilter` property. Without explicitly setting that to `true`, the new props won't show at all. So one might wonder why these props are not being displayed despite setting them. By grouping them via some name prefix like `optionFilter`, this makes it more clear.